### ISSUE TITLE
User "John455a7" reported problem with PA Frames and Vertibirds

### ISSUE THREAD

**Issue by Scrivener07** [Scrivener07/FO4_Overlays#23]
```
I dont know if everyone knows about this but i found out that, when your HUD overlay is gone 
because you used a vertibird or similar, then just enter a power armor Frame 
(Not neccessarely one with a helmet) with your ODST helmet on and then leave it. 
Hud overlay should be Back and working. This Little Workaround seems to be mentioned 
nowhere. Could be a sticky on the Nexuspage of scrivener07 .

I dont know, but would it be possible to create some sort of fix , that… sort of activates the effect from entering and leaving a PA Frame whenever leaving a vertibird?
So that the HUD overlay would come back automatically?

I see, i just can say that me and some other users at some reddit post (dont know exactly where) 
had this and the only way to fix the vanishing of the overlay after vertibird travel was enter&exit 
power armor with odst helmet on. It happens with 100% reliability and the PA Workaround also 
never Fails... Dont know what it is...

I mentioned the Workaround here in case someone suffers from the same issue.
Drove me nuts until i read the reddit post...
Now, everytime after using a vertibird, i fetch my PA via Amazing Follower Tweaks and enter and Exit. Not ideal, but it gets the Job done...

All i can say so far, is that the Overlay is stable and works all the time for me, but after using a 
vertibird, no matter if one from a mod or the vanilla BoS one, called via smokegrenade, the 
Overlay is gone and flickers shortly up when entering a new cell or load the game. 
But after entering a PA Frame, no matter which one, with the ODST Helmet on,
everything is back to normal and 100% reliable.
```

**Comment by Scrivener07** [comment]
Check these out to ease the testing of vertibirds.
- https://www.nexusmods.com/fallout4/mods/12312
- https://www.nexusmods.com/fallout4/mods/25565

**Comment by Scrivener07** [comment]
This is a video of the bug. https://youtu.be/xGbeVHbypp0

**Comment by Scrivener07** [comment]
This is a console batch file that allows instant testing of vertibirds in game. Add to `Data\EnableVertibirds.txt`. Run the console command in game. `bat EnableVertibirds`

[EnableVertibirds.txt](https://github.com/Scrivener07/FO4_Overlays/files/2948097/EnableVertibirds.txt)
```
Set BoSFastTravelCanUse To 1
Set BoSFastTravelPilot To 1
PRID 1BBC91
Enable
Player.AddItem 00056917 20
```

**Comment by Scrivener07** [comment]
This forum post is highly related to this bug https://forums.nexusmods.com/index.php?/topic/6672312-isfirstperson-no-longer-works-as-a-variablebool/

Thanks for the insights shavkacagarikia and others.

**Comment by Scrivener07** [comment]
This has been fixed as of commit 0bde36b. The fixes are included with version [1.2.0](https://github.com/Scrivener07/FO4_Overlays/releases/tag/v1.2.0).
